src/compas_timber/_fabrication/btlx_process.py

```diff
@@ -148,3 +148,44 @@ class LimitationTopType(object):
     LIMITED = "limited"
     UNLIMITED = "unlimited"
     POCKET = "pocket"
+
+
+class MachiningLimits(object):
+    """Configuration class for the machining limits of the cut.
+
+    Attributes
+    ----------
+    EXPECTED_KEYS : set
+        The expected keys for the limits dictionary.
+    face_limited_start : bool
+        Limit the start face.
+    face_limited_end : bool
+        Limit the end face.
+    face_limited_front : bool
+        Limit the front face.
+    face_limited_back : bool
+        Limit the back face.
+
+    Properties
+    ----------
+    limits : dict
+        The limits dictionary with values as a boolean.
+    """
+
+    EXPECTED_KEYS = ["FaceLimitedStart", "FaceLimitedEnd", "FaceLimitedFront", "FaceLimitedBack"]
```

Comment:
what is this for? is it used anywhere?

Reply:
Yes i am using it in the setter of the machining limits of the Lap object where I check if the parameter provided is a dictionary and after that i check if both the keys and values are what they should.